9c-internal/9c-claim-items-test/values.yaml

```diff
@@ -197,3 +197,19 @@ patrolRewardService:
 
   db:
     local: true
+
+rudolfService:
+  image:
+    tag: "git-64ad20a7abb298b1b4e1fdeca00be20780c14e91"
+
+  config:
+    ncgMinter: "0x4fa78af2c9fb3391ef05f1f1f8fe9565137a00f9"
+    graphqlEndpoint: "http://k8s-9cclaimi-remotehe-a491fa26e4-b6e7e66fcd3d3e9a.elb.us-east-2.amazonaws.com/graphql"
+  kms:
+    keyId: "54436222-3b06-4ddb-b661-f2cd54456893"
+    publicKey: "04ff006e2434dc04000971395e5e47012e4ec7570dfbbb87a02e4b12d33ec0c6ec329fdba089f7b5bfce7b8cbcdf3f9e662fade6a63066a9b1e17429687fbdb9de"
+
+  serviceAccount:
+    roleArn: "arn:aws:iam::319679068466:user/internal-9c-rudolf-signer"
```

Comment:
i dont think service account can use user IAM. it's good to try but maybe we need IAM role instead

Reply:
I replaced it with role arn.